### PR TITLE
Fix keybinds/instructions and setup for future Firefox commands configuration

### PR DESCRIPTION
**Pull request by jwoos**
Refer to #21 

I decided to check whether `browser.commands.update` exists. If it does, it has to be Firefox. If it doesn't, it's Chrome. This is easy since we only support these two at the moment but might want to find a more robust solution. Or maybe the browsers might support the API sometimes in the future?

### CODE REVIEW

File: src/scripts/popup/index.ts [hunk 1]
```diff
@@ -10,8 +10,21 @@ import {browser} from 'webextension-polyfill-ts';
...
-	port.postMessage(tabbo.PopUpCommand.KEYBINDS);
+utils.queryOrThrow('#keybinds').addEventListener('click', async (): void => {
+	/* FIXME
+	 * This is a pretty hacky way of detecting browser. No other browser other than Firefox
+	 * supports `browser.commands.update` as of 02/15/19 so check if it's available.
+	 * If it isn't, then it's probably Chrome as we only package the extension for Chrome and
+	 * Firefox as of now. Chrome also doesn't support `browser.runtime.getBrowserInfo` so
+	 * we can't use that!
+	 */
+	if (browser.commands.update === null) {
```
Comment (dqgorelick) [inline src/scripts/popup/index.ts]: Hackity hack hack 🙃

**Comment by jwoos** [comment]
Updated to use `navigator.userAgent` to check for browser!

File: src/scripts/popup/index.ts [hunk 2]
```diff
@@ -10,8 +10,23 @@ import {browser} from 'webextension-polyfill-ts';
...
 
-utils.queryOrThrow('#keybinds').addEventListener('click', (): void => {
-	port.postMessage(tabbo.PopUpCommand.KEYBINDS);
+utils.queryOrThrow('#keybinds').addEventListener('click', async (): void => {
+	/* FIXME
+	 * Chrome doesn't support `browser.runtime.getBrowserInfo` so we can't use that!
+	 * When it does, change to that.
+	 */
+	const browserType: utils.Browser = utils.checkBrowser();
+	alert(browserType);
```
Comment (dqgorelick) [inline src/scripts/popup/index.ts]: can we please remove this before merging?

**Comment by dqgorelick** [comment]
I'd just like to add this back: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/captureVisibleTab 

once it's added we're good to merge! I understand this is a dev version for firefox, but I'd like the chrome version to be without any bugs / missing features if we can :)

File: src/scripts/popup/index.ts (hunk 2 above)
Comment (jwoos) [reply src/scripts/popup/index.ts]: oops

**Comment by jwoos** [comment]
Actually, I think it's better to add that in a separate PR. Just to keep things focused!

**Pull request merged**